src/pytorch_lightning/core/module.py

```diff
@@ -49,12 +49,20 @@
 from pytorch_lightning.utilities.parsing import collect_init_args
 from pytorch_lightning.utilities.rank_zero import rank_zero_debug, rank_zero_deprecation, rank_zero_warn
 from pytorch_lightning.utilities.signature_utils import is_param_in_hook_signature
-from pytorch_lightning.utilities.types import _METRIC_COLLECTION, EPOCH_OUTPUT, LRSchedulerTypeUnion, STEP_OUTPUT
+from pytorch_lightning.utilities.types import (
+    _METRIC_COLLECTION,
+    EPOCH_OUTPUT,
+    LRSchedulerPLType,
+    LRSchedulerTypeUnion,
+    STEP_OUTPUT,
+)
 from pytorch_lightning.utilities.warnings import WarningCache
 
 warning_cache = WarningCache()
 log = logging.getLogger(__name__)
 
+MODULE_OPTIMIZERS = Union[Optimizer, LightningOptimizer, List[Optimizer], List[LightningOptimizer]]
```

Comment:
added to correct ln 134, 137, 149

Reply:
Perhaps this should go into the types utility module? Not sure. Let's ask @carmocca for opinion.

Reply:
If we see an use anywhere other than this file, I'd say yes.

Reply:
let's keep it in `types.py` anyway. It might be hard to find if we need it again for other modules in the future.

Reply:
I couldn't do it because of circular imports.